package.json

```diff
@@ -1,12 +1,16 @@
 {
-  "name": "konvoy-website",
+  "name": "kuma-website",
   "description": "The website and docs for Konvoy.",
```

Comment:
Let's replace `Konvoy` in the `description`

Reply:
@jakubdyszkiewicz and I talked about having a directory for the most recent features and changes that isn't yet linked to publicly on the website. So it would be the starting point where changes are made, and then it's cut to a new release with this script.

But we could instead copy the latest and work from that. That would be an easy enough change. @jakubdyszkiewicz what are your thoughts?

Reply:
Ah, it wasn't clear to me that I'm expected to edit files inside `draft` folder.

Then, I have another problem. I've made some changes inside `draft` folder and now I need to preview them.

When I start the app using `yarn docs:dev` and open `http://localhost:8080/docs/draft/`, there is no navigation tree on the left.

Can preview `draft` version without creating a release ?

Reply:
Ah I overlooked the sidebar setup for the draft version. My apologies for being unclear. I'm going to revise the readme as well to convey the overall flow.